cluster/images/conformance/Makefile

```diff
@@ -31,7 +31,15 @@ E2E_GO_RUNNER_BIN?=$(shell test -f $(LOCAL_OUTPUT_PATH)/go-runner && echo $(LOCA
 
 CLUSTER_DIR?=$(shell pwd)/../../../cluster/
 
-BASEIMAGE=debian:stretch-slim
+
+ifeq ($(ARCH),arm64)
+BASEIMAGE=arm64v8/debian:stretch-slim
+endif
+ifeq ($(ARCH),ppc64le)
+BASEIMAGE=ppc64le/debian:stretch-slim
+endif
+# amd64
+BASEIMAGE?=debian:stretch-slim
```

Comment:
hello, i don't think we should pin a couple of architectures in the Makefile like so.
instead we could make the BASEIMAGE pluggable on Makfile invocation.

so something like this:
```
BASEIMAGE?=debian:stretch-slim
```

and then one can call:
```
make BASEIMAGE=arm64v8/debian:stretch-slim
```

Reply:
I agree that would be nice. there is a BASEIMAGE file based approach used under test/images/image-uti.sh. 

However this is the cluster/images tree. I copied this approach from etcd sibling image, which uses if conditions inside the makefile as well. 

Is it okay if I do the cleanup you suggested for *both* then in a separate commit? I am fairly new to this and don't fully understand where the invocation of the Makefile is triggered in order to pass in the right base image from the caller.

Reply:
ideally, we shouldn't touch the etcd image as it has different maintainers and different usage.

> The debian base image is not multi-arch, so we need to select
the architecture specific image based on which ARCH we currently
build for.

if this is true i'm confused what is the existing behavior.

as per the readme and makefile for conformance, were we just always building for amd64 and just tagging the image for different architectures and pushing it to GCR?

if that is the case it might be better to use the method you proposed, but also:
- include all supported architectures amd64 arm arm64 ppc64le s390x
- depending on ARCH passed to the makefile pick the correct debian baseimage
- update the readme to include build steps for all architectures